src/orders/components/OrderCardTitle/OrderCardTitle.tsx

```diff
@@ -102,11 +114,11 @@ const selectStatus = (status: CardTitleStatus) => {
     case FulfillmentStatus.CANCELED:
       return StatusType.ERROR;
```

Comment:
```suggestion
```

Reply:
I think it's better to leave it verbose as it is, in case if the color of canceled status is changed in the future.